### ISSUE TITLE
LazyLoad Module Causes External jQuery Render-Blocking Issue

### ISSUE THREAD

**Issue by ghost** [wp-media/wp-rocket#4194]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version: **3.9.0.5**
- Used the search feature to ensure that the bug hasn’t been reported before: **Yep, nothing found.**

**Describe the Bug:**

When WPR's LazyLoad feature is activated, it causes jQuery (when loaded externally) to be render-blocking. In our case, when using https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js

We attempted to fix the issue using all options offered under WPR's File Optimization > Javascript Files section, but none of them worked. 
 
**To Reproduce:**

(1) Load jQuery externally. 
(2) Clear all cache.
(3) Activate WPR LazyLoad (all options).
(4) Clear all cache.
(5) Run Lighthouse, PageSpeed, and/or webpagetest.
(6) The following message will appear: "Eliminate render-blocking resources" (Lighthouse, PageSpeed), or a render-blocking symbol (orange X) will appear next to the jQuery file (webpagetest).

**Expected Behavior:**

Fix our issue using WPR.

**Useful Information (Fix):**

We researched this issue quite a bit. We ran into [this article](https://onlinemediamasters.com/wp-rocket-settings/), which offered a solution to the above based on the following statement:

> **Load JavaScript Deferred** – loads JavaScript after the page has finished parsing. It’s one of the best ways to eliminate render-blocking resources (in PSI), but I found WP Rocket doesn’t always do a great job. If you still see render-blocking issues, try installing Autoptimize and Async JavaScript on top of WP Rocket which do a better job optimizing CSS and JavaScript. If you optimize CSS and JS in Autoptimize, it will turn off minify/combine CSS + JS in WP Rocket and lets Autoptimize handle it. This is how I went from 7 render-blocking resources to only 1.

We applied the recommended solution and it worked. Yes, we cleared cache and re-tested (all good). The solution involved installing and activating the plugin, [Async Javascript](https://wordpress.org/plugins/async-javascript/), and applying [these plugin settings](https://prnt.sc/1ehucqq) (**Asyn Javascript Method:** Defer, **jQuery Method:** Exclude)

**Bottom Line:**

We shouldn't have to rely on another plugin to fix this issue. Recommend offering the same options that Async Javacript does and/or revise WPR's code -- as necessary.

**Comment by ghost** [comment]
**Update:**

Above occurs when the noted asset, CDNJS jQuery, is preloaded in the header and WPR's JS modules are activated.

We just learned, however, preloading the asset is not recommended. Instead, we prefetched the asset. Issue fixed, but recommend Team WP Rocket investigate above conflict.

Cheers!
